page/index.html

```diff
@@ -1,99 +1,158 @@
 <!doctype html>
 <html class="no-js" lang="">
+
 <head>
 	<meta charset="UTF-8">
 	<meta http-equiv="x-ua-compatible" content="ie=edge">
 	<title>presentation-fundementals</title>
 	<meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<link href='https://fonts.googleapis.com/css?family=Oswald' rel='stylesheet' type='text/css'>
 	<link href='https://fonts.googleapis.com/css?family=Lato:400,300,200,100' rel='stylesheet' type='text/css'>
 	<link href='https://fonts.googleapis.com/css?family=Merriweather:400,300,200,100' rel='stylesheet' type='text/css'>
-	<link rel="stylesheet" type="text/css" href="http://www.google.com/fonts/specimen/Lora">
+	<link href='https://fonts.googleapis.com/css?family=Lora:900,400,400italic' rel='stylesheet' type='text/css'>
 	<link rel="stylesheet" href="style.css">
 </head>
+
 <body>
 	<header>
 		<h1>CSS OFF</h1>
 	</header>
-	<article>
+	<article class="box-model">
 		<figure>
 			<img src="../images/barbie.png" alt="">
 		</figure>
 		<h1>Box Model</h1>
 		<section>The Barbie doll was invented in 1959 by Ruth Handler (co-founder of Mattel), whose own daughter was called Barbara. Barbie was introduced to the world at the American Toy Fair in New York City. Barbie's job was teenage fashion doll. The full name fo the first doll was Barbie Millicent Roberts, from Willows, Wisconsin.</section>
 	</article>
-	<article>
-		<figure>
-			<img src="../images/Photo.png" alt="">
-		</figure>
+	<article class="positioning-relative">
+		<div class="volume-number">VOLUME 3 - SEPTEMBER 2013</div>
```

Comment:
I would argue that this could be an `<h3>` as this appears to be some important content. Welcome to the SEO debate.